internal/cli/atlas/privateendpoints/datalake/aws/delete.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DeleteOpts struct {
+	cli.GlobalOpts
+	*cli.DeleteOpts
+	store store.DataLakePrivateEndpointDeleter
+}
+
+func (opts *DeleteOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *DeleteOpts) Run() error {
+	return opts.Delete(opts.store.DataLakeDeletePrivateEndpoint, opts.ConfigProjectID())
+}
+
+// mongocli atlas privateEndpoint(s) delete <ID>> --projectId projectId.
+func DeleteBuilder() *cobra.Command {
+	opts := &DeleteOpts{
+		DeleteOpts: cli.NewDeleteOpts("Private endpoint '%s' deleted\n", "Private endpoint not deleted"),
+	}
+	cmd := &cobra.Command{
+		Use:     "delete <ID>",
+		Aliases: []string{"rm"},
+		Short:   "Delete a specific Data Lake private endpoint for your project.",
+		Annotations: map[string]string{
+			"args":      "entry",
```

Comment:
sorry, missed one question - is there a reason for us to choose call the arg entry instead of private endpoint? should I do the same in my PR to make this aligned across commands?

Reply:
I believe we have always called the args in this was for delete commands. This is something that the doc may know

Reply:
value here should match the valu on the usage, also docs prefer to be specific about the name of these args
```suggestion
			"args":      "privateEndpointId",
```

Reply:
ack, I will update other delete commands too